tests/parsely/test-mu-parsely-integration.php

```diff
@@ -70,7 +72,7 @@ public function test_parsely_class_existance() {
 
 		maybe_load_plugin();
 
-		if ( 'disabled' !== self::$test_mode ) {
+		if ( ! $this->is_plugin_disabled() ) {
```

Comment:
Apart from simple disabled mode now we have also filter and option values that represents disabled mode so added a function that checks for each possible config through which plugin can be disabled.